MASH-FRET/source/simulations/arb2phtn.m

```diff
@@ -1,25 +1,11 @@
-function dat = arb2phtn(varargin)
+function dat = arb2phtn(dat,offset,K,eta)
 
 % Created the 23rd of April 2014 by M�lodie C.A.S Hadzic
-% Last update: 14th of March 2018 by Richard B�rner
+% Last update: 12th of March 2019 by M�lodie Hadzic
+% >> obligates to give offset, K and eta values in input
 %
-% Comments adapted for Boerner et al 2017
-
-dat = varargin{1};
-
-if size(varargin,2)==3
-    K = varargin{2};
-    eta = varargin{3};
-elseif size(varargin,2)==2
-    eta = varargin{2};
-    K = 57.7;
-else
-    eta = 0.95;
-    K = 57.7;
-end
-
-offset = 0; % offset implemented in camera noise model. Do not change!
```

Comment:
I try to be as precise as possible:
1. "none" No camera noise: PC = EC = IC. A conversion is basically not necessary. The offset is acutally given in photons. The offset has no noise. 
2. "P" Poisson noise: The offset has no noise. The offset is actually given in EC. 
3. "N" Gaussian noise: The offset has no noise. The offset is given in EC.
4. "NexpN": The offset has no noise. The offset is given in EC. But the zero photon value is noisy. Offset is necessary to avoid negative values. This will be the case for the arb2phtn conversion after substracting the offset befor converting IC -> PC
4. "PGN": The offset has no noise. The offset is given in EC. But the zero photon value is noisy. Offset is necessary to avoid negative values. This will be the case for the arb2phtn conversion after substracting the offset befor converting IC -> PC

Reply:
I think we will need to go through this via Skype, whenever you have time